features/admin.tenants.v1/components/dropdown/tenant-dropdown.tsx

```diff
@@ -638,7 +684,10 @@ const TenantDropdown: FunctionComponent<TenantDropdownInterface> = (props: Tenan
                                                         organizationType === OrganizationType.SUBORGANIZATION
                                                             ? organizationName
                                                             : tenantAssociations
-                                                                ? tenantAssociations.currentTenant
+                                                                ? tenantAssociations.currentTenant?.domain +
```

Comment:
With this new logic, we have introduced nested ternary expresssions several levels deep. Shall we refactor this  to a separate function instead?